File: C/exercicio-05/solucao.c
#include <stdio.h>

void calcularMedia() {
  float prova1, prova2, media;

  printf("Digite a nova da prova 1: " );
  scanf("%f", &prova1);

  printf("Digite a nova da prova 2: " );
  scanf("%f", &prova2);

  media = (prova1 + prova2) / 2;

  printf("A nota da %f com a %f equivale a %f", prova1, prova2, media);

}

int main() {
  calcularMedia();
  return 0;
}

File: Python/exercicio19/solucao.py
'''
19) Crie um algoritmo que leia o nome e as duas notas de um aluno, calcule a sua
média e mostre na tela. No final, analise a média e mostre se o aluno teve ou
não um bom aproveitamento (se ficou acima da média 7.0). 
'''

def main():
    def avaliar_nota():
        prova1 = float(input("Digite a nota da primeira prova: "))
        prova2 = float(input("Digite a nota da segunda prova: "))
        media = (prova1 + prova2) / 2

        print(f'A média das provas é: {media}')

        print("Aprovado !") if media >= 7 else print("Reprovado !") 

    avaliar_nota()


main()